xml/operations-maintenance-deploy-ptf.xml

```diff
@@ -166,19 +159,61 @@ drwxr-xr-x 10 root root 4096 Aug 14 16:13 nova-20180814T161306Z</screen>
    <para>
     Run the update
    </para>
-   <screen>&prompt.ardana;cd ~/scratch/ansible/next/ardana/ansible
-&prompt.ardana;ansible-playbook -i hosts/verb_hosts ardana-update.yml --limit  comp0001-mgmt</screen>
+<screen>&prompt.ardana;cd ~/scratch/ansible/next/ardana/ansible
+&prompt.ardana;ansible-playbook -i hosts/verb_hosts ardana-update.yml</screen>
    <para>
     When it has finished, you can see that an additional virtual environment
-    has been installed
+    has been installed.
    </para>
-   <screen>&prompt.ardana;ls -la /opt/stack/venv
+<screen>&prompt.ardana;ls -la /opt/stack/venv
 total 28
 drwxr-xr-x  9 root root 4096 Jul 18 15:47 neutron-20180718T154642Z
 drwxr-xr-x  9 root root 4096 Aug 14 16:13 neutron-20180814T161306Z
 drwxr-xr-x  9 root root 4096 Oct 30 12:43 neutron-20181030T124310Z &lt;&lt;&lt; New venv installed
 drwxr-xr-x 10 root root 4096 May 28 09:30 nova-20180528T092954Z
    drwxr-xr-x 10 root root 4096 Aug 14 16:13 nova-20180814T161306Z</screen>
   </step>
+  <step>
+   <para>
+    The PTF may also have <literal>RPM</literal> package updates in addition to
```

Comment:
rather than replicating all of that, can we point to the generic "here's how you deploy updates" section? this is probably already documented elsewhere.